static/app/utils/replays/replayReader.tsx

```diff
@@ -61,6 +61,32 @@ type RequiredNotNull<T> = {
 
 const sortFrames = (a, b) => a.timestampMs - b.timestampMs;
 
+function removeDuplicateClicks(frames: BreadcrumbFrame[]) {
+  const slowClickFrames = frames.filter(
+    frame => frame.category === 'ui.slowClickDetected'
+  );
+
+  const clickFrames = frames.filter(frame => frame.category === 'ui.click');
+
+  const noClickFrames = frames.filter(
```

Comment:
the name matches the pattern of the other two, but it might read better as like `const otherFrames`